src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY_FACTOR = 2;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
```

Comment:
Here we would use
```suggestion
        if (size >= (int) (table.length * DEFAULT_LOAD_FACTOR)) {
```
and that's it, never used again